File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/d/SkinTestParamDTO.java
package iih.ci.ord.dto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 皮试逻辑入口参数DTO DTO数据 
 * 
 */
public class SkinTestParamDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 所属组织
	 * @return String
	 */
	public String getId_org() {
		return ((String) getAttrVal("Id_org"));
	}
	/**
	 * 所属组织
	 * @param Id_org
	 */
	public void setId_org(String Id_org) {
		setAttrVal("Id_org", Id_org);
	}
	/**
	 * 患者
	 * @return String
	 */
	public String getId_pi() {
		return ((String) getAttrVal("Id_pi"));
	}
	/**
	 * 患者
	 * @param Id_pi
	 */
	public void setId_pi(String Id_pi) {
		setAttrVal("Id_pi", Id_pi);
	}
	/**
	 * 出生日期
	 * @return FDate
	 */
	public FDate getDt_birth() {
		return ((FDate) getAttrVal("Dt_birth"));
	}
	/**
	 * 出生日期
	 * @param Dt_birth
	 */
	public void setDt_birth(FDate Dt_birth) {
		setAttrVal("Dt_birth", Dt_birth);
	}
	/**
	 * 用药服务
	 * @return String
	 */
	public String getId_srv() {
		return ((String) getAttrVal("Id_srv"));
	}
	/**
	 * 用药服务
	 * @param Id_srv
	 */
	public void setId_srv(String Id_srv) {
		setAttrVal("Id_srv", Id_srv);
	}
	/**
	 * 用药
	 * @return String
	 */
	public String getId_mm() {
		return ((String) getAttrVal("Id_mm"));
	}
	/**
	 * 用药
	 * @param Id_mm
	 */
	public void setId_mm(String Id_mm) {
		setAttrVal("Id_mm", Id_mm);
	}
	/**
	 * 皮试服务
	 * @return String
	 */
	public String getId_skinsrv() {
		return ((String) getAttrVal("Id_skinsrv"));
	}
	/**
	 * 皮试服务
	 * @param Id_skinsrv
	 */
	public void setId_skinsrv(String Id_skinsrv) {
		setAttrVal("Id_skinsrv", Id_skinsrv);
	}
	/**
	 * 皮试药品
	 * @return String
	 */
	public String getId_skinmm() {
		return ((String) getAttrVal("Id_skinmm"));
	}
	/**
	 * 皮试药品
	 * @param Id_skinmm
	 */
	public void setId_skinmm(String Id_skinmm) {
		setAttrVal("Id_skinmm", Id_skinmm);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/d/SkinTestParamDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.d
{
    /// <summary>
    /// SkinTestParamDTO 的摘要说明。
    /// </summary>
    public class SkinTestParamDTO : BaseDTO {

        public SkinTestParamDTO() {
 
        }

        /// <summary>
        /// 所属组织
        /// </summary>
		public string Id_org {
            get { return getAttrVal<string>("Id_org",null); }
            set { setAttrVal<string>("Id_org", value); }
        }

        /// <summary>
        /// 患者
        /// </summary>
		public string Id_pi {
            get { return getAttrVal<string>("Id_pi",null); }
            set { setAttrVal<string>("Id_pi", value); }
        }

        /// <summary>
        /// 出生日期
        /// </summary>
		public DateTime? Dt_birth {
            get { return getAttrVal<FDate>("Dt_birth",null); }
            set { setAttrVal<FDate>("Dt_birth", value); }
        }

        /// <summary>
        /// 用药服务
        /// </summary>
		public string Id_srv {
            get { return getAttrVal<string>("Id_srv",null); }
            set { setAttrVal<string>("Id_srv", value); }
        }

        /// <summary>
        /// 用药
        /// </summary>
		public string Id_mm {
            get { return getAttrVal<string>("Id_mm",null); }
            set { setAttrVal<string>("Id_mm", value); }
        }

        /// <summary>
        /// 皮试服务
        /// </summary>
		public string Id_skinsrv {
            get { return getAttrVal<string>("Id_skinsrv",null); }
            set { setAttrVal<string>("Id_skinsrv", value); }
        }

        /// <summary>
        /// 皮试药品
        /// </summary>
		public string Id_skinmm {
            get { return getAttrVal<string>("Id_skinmm",null); }
            set { setAttrVal<string>("Id_skinmm", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_org", "Id_pi", "Dt_birth", "Id_srv", "Id_mm", "Id_skinsrv", "Id_skinmm"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.d.SkinTestParamDTO";
        }
    }
}
